components/application-registry/internal/metadata/specification/assetstore/docstopicrepository.go

```diff
@@ -107,14 +110,17 @@ func (r repository) create(docsTopic v1alpha1.DocsTopic) apperrors.AppError {
 	return nil
 }
 
-func (r repository) update(docsTopic v1alpha1.DocsTopic) apperrors.AppError {
+func (r repository) update(docsTopic v1alpha1.ClusterDocsTopic) apperrors.AppError {
 	u, err := toUstructured(docsTopic)
 	if err != nil {
 		return err
 	}
 
 	{
-		_, err := r.resourceInterface.Update(u, metav1.UpdateOptions{})
+		err := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
According to @aszecowka comment https://github.com/kyma-project/kyma/pull/3507#discussion_r272075860 I think it might not be the correct way of using `RetryOnConflict`.

Reply:
Done.